tracer/src/Datadog.Trace/Configuration/FeaturesProduct.cs

```diff
@@ -0,0 +1,12 @@
+// <copyright file="FeaturesProduct.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.RemoteConfigurationManagement;
+
+namespace Datadog.Trace.Configuration;
```

Comment:
does it make sense to have them in `Configuration` folder rather than in a RemoteConfiguration folder or in `RemoteConfigurationManagement` folder?  Just thought that the `Configuration` folder is more for local config? but not a blocker ofc